ports/jwt-cpp/vcpkg.json

```
@@ -0,0 +1,9 @@
+{
+  "name": "jwt-cpp",
+  "version-string": "0.5.0",
```

```suggestion
  "version-semver": "0.5.0",
```
For package versions. it would be more proper to use `version-semver` here. Please see https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions